docs/translating/polish.md

```diff
@@ -2,6 +2,10 @@
 
 Use semi-formal Polish for translation, some specifics:
 
+* In polish websites it's more common to use just "search" ("szukaj") than, "search for people".  
```

Comment:
I'm not sure what it's referring to -- the sidebar with the user names? I think I would use "szukaj/znajdź osoby/ę", because it's not a general search.